assets/js/booking.js

```diff
@@ -16,7 +16,72 @@
         });
     });
 
+    var validate = function(){
+        var from_d = $('input#from_date');
+        _missing(from_d, 'Need a From date');
```

Comment:
@bobbyjoe  is this message seen by a user?  if so, "Please enter Check In date" sounds more friendly.